README.md

```diff
@@ -32,26 +32,28 @@ Optimus has two components, Optimus service that is the core orchestrator instal
 $ brew install odpf/taps/optimus
 $ optimus --help
 
-optimus v0.0.2-alpha.1
-
-optimus is a scaffolding tool for creating transformation job specs
+Optimus is an easy-to-use, reliable, and performant workflow orchestrator for
+data transformation, data modeling, pipelines, and data quality management.
 
 Usage:
   optimus [command]
 
 Available Commands:
+  backup      Backup a resource and its downstream
+  completion  generate the autocompletion script for the specified shell
```

Comment:
Generate supposed to be capital?

Reply:
`completion` command is not added by us and is added by the cobra library so the description is pre-generated. Didn't find an easy way to modify it so I think this looks good enough for now?

Reply:
yes 👍